From c2992046db296d402d5a009bcfb2ea0145aa1272 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 4 Sep 2025 15:37:32 +0800
Subject: [PATCH 083/101] ACPI: processor_idle: Fix build with CONFIG_WERROR

When WERROR is enabled, build fails with the following error:

  drivers/acpi/processor_idle.c:699:39: error: suggest braces around empty body in an 'if' statement [-Werror=empty-body]
    699 |                 ACPI_FLUSH_CPU_CACHE();
        |                                       ^
  cc1: all warnings being treated as errors

Add a bracket around `if (cx->type == ACPI_STATE_C3)' to fix build.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 drivers/acpi/processor_idle.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/acpi/processor_idle.c b/drivers/acpi/processor_idle.c
index 81f372c64074..f400a78b29b7 100644
--- a/drivers/acpi/processor_idle.c
+++ b/drivers/acpi/processor_idle.c
@@ -695,8 +695,9 @@ static int __cpuidle acpi_idle_enter(struct cpuidle_device *dev,
 		}
 	}
 
-	if (cx->type == ACPI_STATE_C3)
-		ACPI_FLUSH_CPU_CACHE();
+	if (cx->type == ACPI_STATE_C3) {
+		ACPI_FLUSH_CPU_CACHE()
+	}
 
 	acpi_idle_do_entry(cx);
 
-- 
2.25.1

